pages/linux/etckeeper.md

```diff
@@ -0,0 +1,24 @@
+# etckeeper
+
+> Track system configuration files in git.
+> More information: http://etckeeper.branchable.com/.
+
+- First time setup (run from /etc):
```

Comment:
How about - "Initialize a git repo (run from /etc)"

Reply:
It does a bit more than that though --  here's the relevant documentation:

> The etckeeper init command initialises an /etc/.git/ repository. If you installed etckeeper from a package, this was probably automatically performed during the package installation. If not, your first step is to run it by hand:
>
>`etckeeper init`
>
>The etckeeper init command is careful to never overwrite existing files or directories in /etc. It will create a .gitignore if one doesn't already exist (or update content inside a "managed by etckeeper" comment block), sets up pre-commit hooks if they don't already exist, and so on. It does not commit any files, but does git add all interesting files for an initial commit later.
>
>Now you might want to run git status to check that it includes all the right files, and none of the wrong files. And you can edit the .gitignore and so forth. Once you're ready, it's time to commit:

So it creates an empty repo, sets up a gitignore, sets up various hooks, and adds interesting files for the first commit. `git init` just creates an empty repo; would that be a bit confusing?

Reply:
I see. How about - "Perform various housekeeping tasks to setup a git repo (run from /etc)" ? I think that gives an indication that basically it sets up a git repo and performs other tasks.

Reply:
Sounds great! New commit inbound.